omnipod/src/main/res/layout/omnipod_overview.xml

```diff
@@ -779,63 +779,77 @@
-                    android:drawableTop="@drawable/ic_danarstats"
+                    android:drawableTop="@drawable/ic_access_alarm"
                     android:paddingLeft="0dp"
                     android:paddingRight="0dp"
-                    android:text="@string/omnipod_overview_button_riley_link_stats" />
+                    android:text="@string/omnipod_overview_button_set_time"
+                    android:visibility="gone" />
 
                 <Button
-                    android:id="@+id/omnipod_overview_button_pulse_log"
+                    android:id="@+id/omnipod_overview_button_resume_delivery"
                     style="@style/ButtonSmallFontStyle"
                     android:layout_width="match_parent"
                     android:layout_height="match_parent"
                     android:layout_weight="1"
-                    android:drawableTop="@drawable/ic_cp_bolus_correction"
+                    android:drawableTop="@drawable/ic_local_activate"
                     android:paddingLeft="0dp"
                     android:paddingRight="0dp"
-                    android:text="@string/omnipod_overview_button_read_pulse_log"
+                    android:text="@string/omnipod_overview_button_resume_delivery"
                     android:visibility="gone" />
 
                 <Button
-                    android:id="@+id/omnipod_overview_button_set_time"
+                    android:id="@+id/omnipod_overview_button_suspend_delivery"
                     style="@style/ButtonSmallFontStyle"
                     android:layout_width="match_parent"
                     android:layout_height="match_parent"
                     android:layout_weight="1"
-                    android:drawableTop="@drawable/ic_access_alarm"
+                    android:drawableTop="@drawable/ic_loop_disabled"
                     android:paddingLeft="0dp"
                     android:paddingRight="0dp"
-                    android:text="@string/omnipod_overview_button_set_time"
+                    android:text="@string/omnipod_overview_button_suspend_delivery"
                     android:visibility="gone" />
 
                 <Button
-                    android:id="@+id/omnipod_overview_button_resume_delivery"
+                    android:id="@+id/omnipod_overview_button_test_beep"
                     style="@style/ButtonSmallFontStyle"
                     android:layout_width="match_parent"
                     android:layout_height="match_parent"
                     android:layout_weight="1"
-                    android:drawableTop="@drawable/ic_local_activate"
+                    android:drawableTop="@drawable/ic_speaker_icon"
                     android:paddingLeft="0dp"
                     android:paddingRight="0dp"
-                    android:text="@string/omnipod_overview_button_resume_delivery"
+                    android:text="@string/omnipod_overview_button_test_beep"
+                     />
```

Comment:
Can you please move the /> to the line above?

Reply:
done